tests/framework/pkg/file/file.go

```diff
@@ -0,0 +1,68 @@
+package file
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// NewDir is a function that creates a directory with given directory name.
+// Ignores the returned error if the directory already exists.
+func NewDir(dirName string) (err error) {
+	err = os.Mkdir(dirName, 0777)
+	if err != nil {
+		if strings.Contains(err.Error(), "file exists") {
+			logrus.Infof("dir already exists: %v", err)
+			return nil
+		} else {
+			return
+		}
+	}
+
+	return
+}
+
+type FileName string
+
+// NewFile is a method that creates a file with the received file name.
+func (f FileName) NewFile(data []byte) (wdPath string, err error) {
+	err = os.WriteFile(string(f), data, 0644)
+	if err != nil {
+		return
+	}
+
+	wdPath, err = f.GetWDFilePath()
+	if err != nil {
+		return
```

Comment:
Should err be returned here?

Reply:
These are called named return values in go, err is re-assigned from its zero value of the named return variable